GlobalBadges/src/components/Settings.tsx

```diff
@@ -129,7 +126,7 @@ export default ({ settings }: SettingsProps) => {
                 <FormSection title="Updates">
                     <FormRow
                         label="Check for Updates on startup"
-                        subLabel={"Checks automatically for updates when starting the Plugin"}
+                        subLabel={"Checks automatically for updates when starting GlobalBadges"}
```

Comment:
Instead of hard coding the name, use `${name}` which is the name from the manifest